runtime/near-vm-logic/README.md

```diff
@@ -1,11 +0,0 @@
-# near-vm-logic
-
-This crate implements the specification of the interface that Near blockchain exposes to the smart contracts.
-It is not dependent on the specific way the smart contract code is executed, e.g. through Wasmer or whatnot, and
-therefore can be used for unit tests in smart contracts.
```

Comment:
Curious this idea never actually materialized to any significant degree.

Reply:
Just for context: https://github.com/near/nearcore/issues/10421